lib/oli/delivery/sections/updates.ex

```diff
@@ -268,12 +270,75 @@ defmodule Oli.Delivery.Sections.Updates do
         "perform_update.MAJOR: section[#{section.slug}] #{Oli.Timing.elapsed(mark) / 1000 / 1000}ms"
       )
 
+      renumber_hierarchy(section)
```

Comment:
Should we add this function call to the `with` clause above? It performs a DB query and might return 
```elixir 
{:ok, query_result} | {:error, Exception.t()}
```